Nouveau Rôle: Loup Feutrer

Commantary opened the pull request:
- Ajout de la classe `RLoupFeutrer`
- Ajout du rôle dans la classe `MainLG`
- Ajout de la carte Loup-Garou pour le Loup Feutrer dans `LGCustomItems`
- Ajout du rôle dans `RoleSort`

**Description**: 
Il gagne avec les **_Loup-Garou_**. 
Au début de la première nuit, il choisit une personne comme exemple. Si la **_Voyante_** regarde le Loup Feutrer, elle verra le rôle de **_l'exemple_**.

Commantary committed:
Nouveau Rôle: Loup Feutrer

- Ajout de la classe RLoupFeutrer
- Ajout du rôle dans la classe MainLG
- Ajout de la carte Loup-Garou pour le Loup Feutrer

Description: Il gagne avec les Loup-Garou. Au début de la première nuit, il choisit une personne comme exemple. Si la voyante regarde le Loup Feutrer, elle verra le rôle de l'exemple.

Nicooow commented:
Mais du coup t'as pas pu faire une vrai icone comme t'as pas accès au ressource pack ?

Nicooow commented:
`    public String getName() {
        return "§c§lLoup Feutrer";
    }`
Tu n'aurais pas oublié de renvoyer le rôle du joueur choisis par le loup ? Car là dans tous les cas la voyante reçoit tout le temps "Loup Feutrer"

Commantary committed:
Fix RVoyante

J'avais oublier de faire en sorte que la Voyante voit le rôle de l'exemple pour le Loup Feutrer

Commantary commented:
> Mais du coup t'as pas pu faire une vrai icone comme t'as pas accès au ressource pack ?

Alors ducoup je change pas les icônes parce que je suis pas graphiste donc ça sera moche et si on y a accès au ressource pack.

> ` public String getName() { return "§c§lLoup Feutrer"; }`
> Tu n'aurais pas oublié de renvoyer le rôle du joueur choisis par le loup ? Car là dans tous les cas la voyante reçoit tout le temps "Loup Feutrer"

J'ai régler ça dans `RVoyante`, je garde en cache la personne qui est l'exemple `.getCache().get("loup_ftr_e")` et si la personne que la _**Voyante**_ sonde est _**Loup Feutrer**_ on récupère le rôle de l'exemple pour le donner à la _**Voyante**_.

Nicooow commented:
Je pense que tu peux faire bien plus propre et sûr que de faire un test sur le getName du rôle, genre direct de la classe

Commantary commented:
> Je pense que tu peux faire bien plus propre

Pour le code en disant bien plus propre, tu veux parler de la syntaxe ?

> sûr que de faire un test sur le getName du rôle, genre direct de la classe

Je n'ai pas compris ce que tu veux dire par là je suis désolé mais je te propose de faire ce dont tu parles et tu le proposer :)

Chaarlesss commented:
L'idée du rôle n'est pas mauvaise en elle-même, cependant son implémentation pose quelques problèmes, notamment car l'idée d'adapter le code de la voyante pour ton rôle est contraire à une convention du plugin: rendre les rôles totalement indépendants du reste du code !
La meilleure solution pour changer le nom affiché pour la voyante serait de créer un nouvel événement qui correspondrait à un "check" de la voyante, et d'ajouter un handler de cet événement dans ton nouveau rôle.

Commantary commented:
Merci Gogume1er je n'avais pas pris ça en compte, je vais voir pour changer cela je t'en remercie

kurtix56 commented:
salut ton rôle est grave intéressant , es-ce qu'avec la maj il est toujours fonctionnel ?

Pioupiou2888 commented:
Salut quand je choisis le loup feutrer comme roles le jeux va pas plus loin qu’apprêt la distribution des rôles .Pouvez vous m'aidez ?
